sw/service-worker.ts

```diff
@@ -9,7 +9,7 @@ import { CacheChildren } from './CacheChildrenStrategy';
 // Setting the default expiration options for the caches.
 const DEFAULT_EXPIRATION_OPTIONS = {
   maxEntries: 128,
-  maxAgeSeconds: 60 * 60 * 24,
+  maxAgeSeconds: 60 * 60 * 24 * 7,   // 7 days, making sure we don't end up with sticky caches.
```

Comment:
just another comment about the fact that we don't have sticky caches because every request updates it.